src/main/java/it/pagopa/ecommerce/payment/methods/utils/PaymentMethodManagementEnum.java

```diff
@@ -0,0 +1,19 @@
+package it.pagopa.ecommerce.payment.methods.utils;
+
+public enum PaymentMethodManagementEnum {
```

Comment:
You can avoid creating a new enum and use the auto-generated one in your code

Reply:
Handled in 27d6dcd